package.json

```diff
@@ -55,18 +56,22 @@
     "angulartics2": "^7.4.1",
     "bcrypt": "^4.0.1",
     "body-parser": "^1.18.3",
+    "axios": "^0.19.2",
```

Comment:
NestJS's `HttpModule` `HttpClient` uses Axios under the hood. The peer dependency guaranteed to work with NestJS is already installed. Replacing instances of Axios with HttpClient will remove the requirement for this dependency.